src/utils/layer-manager-utils.js

```diff
@@ -48,6 +48,8 @@ export const layerManagerOrder = (datasets, activeLayers, callback) => {
   callback({ activeLayers: updatedLayers });
 };
 
+export const layerInMap = (layerTitle, map) => map.layers.items.some(l => l.title === layerTitle);
```

Comment:
I think we can remove that, I haven't seen it used anywhere

Reply:
I'm working on a local branch to change how we are handling layers and I already have updated this `layer-manager-utils` do you think it makes sense to merge this branch as it is and wait for the `load-improvements` branch to handle this dead code?